string ObjectID
string poseID
---
srs_grasping/grasp[] grasps
bool response
---
string fb
